streams/concat_streams_test.ts

```diff
@@ -1,15 +1,15 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
 import { assertEquals, assertRejects } from "../assert/mod.ts";
-import { ConcatStreams } from "./concat_streams.ts";
+import { concatStreams } from "./concat_streams.ts";
 
-Deno.test("new ConcatStreams(streams)", async function () {
+Deno.test("concatStreams(streams)", async function () {
```

Comment:
Nit: consistency with (most) of the rest of `std`. Ditto for other tests.
```suggestion
Deno.test("concatStreams()", async () => {
```